apps/docs/content/guides/guides/components.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Customizing Components
+description: Learn how to customize Hopper components to meet your product needs.
+order: 3
+---
+
+Hopper components are built with well-considered defaults that meet common product needs.
+ However, we understand that product teams often move quickly, iterating at a pace that can move faster than design system updates.
+ As this article on [pace layering](https://bigmedium.com/ideas/design-system-pace-layers-slow-fast.html) explains, it's natural for design systems to evolve more slowly than the products they support.
+ Rather than being a bottleneck, this approach keeps our design system stable and reliable.
+ To support fast-paced product development, Hopper provides a range of options for customization,
+ from simple styling adjustments to creating entirely new components.
+
+## Customization Principles
+
+The Hopper design systems are built on the principle that “components are designed to be overridden, not locked-in.” This allows developers to adjust styles and behaviors to meet specific product needs, while ensuring components remain usable and adaptable.
+
+There are multiple customization methods available, and to highlight why some are better suited than others, I’ve organized them into four tiers:
+
+🥇**Gold Tier:** Maximize reuse of the Design System, applying only minimal overwrites to fit your needs.
+
+🥈**Silver Tier:** Reuse most of the component’s behaviors.
+
+🥉**Bronze Tier:** Reassemble the component’s behavior using smaller utility methods.
+
+🛠️**Do it yourself:** Build the behavior from scratch and use low-level styling tools as needed
+
+## Customization Methods Overview
+
+Here's an overview of the different customization methods:
+
+- 🥇 [Direct styling adjustments](#customization-methods-details--direct-styling-adjustments): You can use CSS Classes or the styling props on existing Design System Components to alter the style as needed.
+
+- 🥇 [Behavior modifications](#customization-methods-details--behavior-modifications): You can use ref, callbacks and controlled properties to modify its behavior.
+
+- 🥈 [Copying component](#customization-methods-details--copying-component): Copy the component code inside your codebase and modify it.
+
+- 🥈 [Custom components using base components](#customization-methods-details--custom-components-using-base-components): Create a new component on top of Design System base components or React Aria Components.
+
+- 🥉 [Custom components using hooks](#customization-methods-details--custom-components-using-hooks): Create a new component on top of Design System hooks or React Aria hooks.
+
+- 🛠️ [Building components from scratch](#customization-methods-details--building-components-from-scratch): Create a component from scratch and leveraging Design Tokens if possible
```

Comment:
```suggestion
- 🛠️ [Building components from scratch](#customization-methods-details--building-components-from-scratch): Create a component from scratch and leverage Design Tokens if possible.
```
Missing period at the end and I think it makes more sense to say leverage to use the same tense.